src/Components/Search/SearchBar.tsx

```diff
@@ -90,7 +91,7 @@ export class SearchBar extends Component<Props, State> {
   private userClickedOnDescendant: boolean
 
   state = {
-    term: "",
+    term: getSearchTerm(window.location),
```

Comment:
Are there any SSR / UI implications to this approach? Meaning: 
- Will it error out (`window is undefined`)
- If there's a `q=` in the url bar, will there be a jump when the client-side mounts?